packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss

```diff
@@ -29,10 +43,57 @@
   margin: 0 $spacing-03;
 }
 
+.toggle {
+  padding: 8px;
+  color: $interactive-01;
+  min-height: 0px;
+
+  &:global(.bx--btn--tertiary) {
+    border-color: $interactive-01;
+  }
+
+  &:global(.bx--tooltip__trigger:active:not([disabled])) {
+    border-color: $interactive-01;
+  }
+
+  &:global(.bx--btn--tertiary:active) {
+    border-color: $interactive-01;
+    background-color: #edf5ff;
```

Comment:
Are these colors defined in our variables?

Reply:
Not this shade of blue. I raised this issue with Ciaran earlier and we've planned to work on standardising the colour palette so we have consistent naming and a comprehensive colour library in our style guide. There's not much I can do until that effort is complete.